hoon/lib/jock.hoon

```diff
@@ -799,12 +992,12 @@
   |=  =tokens
   ^-  @
   ?~  tokens  ~|("expect literal. token: ~" !!)
-  ?.  ?=(?(%literal %symbol) -.i.tokens)
+  ?.  ?=(%literal -.i.tokens)
     ~|("expect literal or symbol. token: {<-.i.tokens>}" !!)
   =/  p=jatom  +.i.tokens
-  ?.  ?=(%number -.p)
+  ?.  ?=(%number -.-.p)
```

Comment:
<-.p

Reply:
I could live with `--+-+` or something but the alternating notation is just absolute wretched